awx/main/models/execution_environments.py

```diff
@@ -44,7 +44,7 @@ class Meta:
     pull = models.CharField(
         max_length=1024,
         choices=PULL_CHOICES,
-        default='missing',
+        default=None,
```

Comment:
I believe you also want to add 
```        
blank=True,
null=True,
```

See here for another example: https://github.com/ansible/awx/blob/devel/awx/main/models/ha.py#L198-L200